integ-test/src/test/java/org/opensearch/sql/calcite/big5/CalcitePPLBig5IT.java

```
@@ -15,6 +15,5 @@ public class CalcitePPLBig5IT extends PPLBig5IT {
   public void init() throws Exception {
     super.init();
     enableCalcite();
-    disallowCalciteFallback();
```

[question] Will allowing fallback in Calcite*IT make the tests loss their detection effect? 

In Calcite*IT, we mean to run the query successfully on Calcite engine. But allowing them fallback will cover up on which engine it's ran successfully in the end. And that disadvantage may affect the development process since developer cannot notice whether the tests fallback or not.